src/satosa/backends/saml2.py

```diff
@@ -181,7 +249,11 @@ def authn_request(self, context, entity_id):
         kwargs = {}
         authn_context = self.construct_requested_authn_context(entity_id)
         if authn_context:
-            kwargs['requested_authn_context'] = authn_context
+            kwargs["requested_authn_context"] = authn_context
+        if self.config.get(SAMLBackend.KEY_MIRROR_FORCE_AUTHN):
```

Comment:
No need to actually check for this; we could always be setting `force_authn` to the returned value by `get_force_authn()`